From 6f7ea879bba03cb98935f28ebd5a83676b18031a Mon Sep 17 00:00:00 2001
From: Koen Vandeputte <koen.vandeputte@ncentric.com>
Date: Thu, 16 Mar 2023 16:18:00 +0530
Subject: [PATCH 041/281] OpenWrt:
 481-mtd-spi-nor-add-support-for-Gigadevice-GD25D05.patch

mtd: spi-nor: add support for Gigadevice GD25D05

Signed-off-by: Koen Vandeputte <koen.vandeputte@ncentric.com>

Change-Id: Id43fcfa7d0b3f24b3522ebe95358d2ed6d252fef
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
---
 drivers/mtd/spi-nor/gigadevice.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mtd/spi-nor/gigadevice.c b/drivers/mtd/spi-nor/gigadevice.c
index d57ddaf1525b..0e75d1e384e0 100644
--- a/drivers/mtd/spi-nor/gigadevice.c
+++ b/drivers/mtd/spi-nor/gigadevice.c
@@ -34,6 +34,10 @@ static const struct spi_nor_fixups gd25q256_fixups = {
 };
 
 static const struct flash_info gigadevice_nor_parts[] = {
+	{ "gd25q05", INFO(0xc84010, 0, 64 * 1024,  1)
+		FLAGS(SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB) 
+		NO_SFDP_FLAGS(SECT_4K | SPI_NOR_DUAL_READ |
+			      SPI_NOR_QUAD_READ) },
 	{ "gd25q16", INFO(0xc84015, 0, 64 * 1024,  32)
 		FLAGS(SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB)
 		NO_SFDP_FLAGS(SECT_4K | SPI_NOR_DUAL_READ |
-- 
2.17.1

